examples/shaders/shaders_julia_set.c

```diff
@@ -93,59 +90,63 @@ int main(void)
         // Update
         //----------------------------------------------------------------------------------
         // Press [1 - 6] to reset c to a point of interest
-        if (IsKeyPressed(KEY_ONE) ||
-            IsKeyPressed(KEY_TWO) ||
-            IsKeyPressed(KEY_THREE) ||
-            IsKeyPressed(KEY_FOUR) ||
-            IsKeyPressed(KEY_FIVE) ||
-            IsKeyPressed(KEY_SIX))
+        for (KeyboardKey k = KEY_ONE; k <= KEY_SIX; ++k)
         {
-            if (IsKeyPressed(KEY_ONE)) c[0] = pointsOfInterest[0][0], c[1] = pointsOfInterest[0][1];
-            else if (IsKeyPressed(KEY_TWO)) c[0] = pointsOfInterest[1][0], c[1] = pointsOfInterest[1][1];
-            else if (IsKeyPressed(KEY_THREE)) c[0] = pointsOfInterest[2][0], c[1] = pointsOfInterest[2][1];
-            else if (IsKeyPressed(KEY_FOUR)) c[0] = pointsOfInterest[3][0], c[1] = pointsOfInterest[3][1];
-            else if (IsKeyPressed(KEY_FIVE)) c[0] = pointsOfInterest[4][0], c[1] = pointsOfInterest[4][1];
-            else if (IsKeyPressed(KEY_SIX)) c[0] = pointsOfInterest[5][0], c[1] = pointsOfInterest[5][1];
-
-            SetShaderValue(shader, cLoc, c, SHADER_UNIFORM_VEC2);
+            if (IsKeyPressed(k))
```

Comment:
Actually for a newby, this code is more complex to understand than previous one... I'd keep the simple version.

Reply:
Thanks, that's fair - reverted in a828267712f514a9a10ad563ede610bf3ee2fb3e